packages/react/src/components/TableOfContents/TOCDesktop.js

```diff
@@ -72,9 +72,17 @@ const TOCDesktop = ({ menuItems, selectedId }) => {
    */
   function triggerFocus(elem) {
     const element = document.querySelector(elem);
```

Comment:
Can use `root` here to prevent `undefined` in SSR environments.

e.g. https://github.com/carbon-design-system/carbon-for-ibm-dotcom/blob/1726663f273fc08429c442b91d061d7a967afb02/packages/react/src/components/Masthead/Masthead.js#L27